bson/build.gradle

```diff
@@ -23,11 +23,10 @@ ext {
 }
 
 jar {
-    manifest {
-        instruction 'Automatic-Module-Name', 'org.mongodb.bson'
-        instruction 'Build-Version', project.gitVersion
-        instruction 'Import-Package',
-                    'javax.xml.bind.*',
-                    'org.slf4j;resolution:=optional'
-    }
+    bnd (
+        '-exportcontents': "*;-noimport:=true",
```

Comment:
Adding this attribute to `bnd()` generates the `Export-Package` attribute in the manifest.

Reply:
Nice